copilot/fastapi/run_tests.sh

```
@@ -0,0 +1,85 @@
+        *)
+        SPECIFIC_TEST="$arg"
+        shift
+        ;;
+    esac
+done
+
+# Run the tests
+if [ -z "$SPECIFIC_TEST" ]; then
+    echo -e "${YELLOW}Running all tests...${NC}"
+    # Add -s to show print outputs which can help with debugging
+    python -m pytest tests/ $VERBOSE $COVERAGE -s
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)